packages/roosterjs-editor-core/lib/corePlugins/CopyPlugin.ts

```diff
@@ -30,6 +32,8 @@ export default class CopyPlugin implements EditorPlugin {
     public dispose() {
         this.copyDisposer();
         this.copyDisposer = null;
+        this.cutDisposer();
+        this.cutDisposer = null;
         this.editor = null;
     }

```

Comment:
Question:

If we do preventDefault() for cut event, will the selected content be removed?

Reply:
Ah, yeah. It does stop that from happening.

Reply:
This will need to be handled (I published this while I was testing my changes).